src/__tests__/ListGroup.spec.js

```diff
@@ -1,73 +1,59 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import { ListGroup } from '..';
+import {
+  testForCustomClass,
+  testForCustomTag,
+  testForDefaultClass,
+} from '../testUtils';
 
 describe('ListGroup', () => {
   it('should render with "list-group" class', () => {
-    const wrapper = shallow(<ListGroup>Yo!</ListGroup>);
-
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('list-group')).toBe(true);
+    testForDefaultClass(ListGroup, 'list-group');
   });
 
   it('should render with "flush"', () => {
-    const wrapper = shallow(<ListGroup flush>Yo!</ListGroup>);
+    render(<ListGroup flush>Yo!</ListGroup>);
 
-    expect(wrapper.text()).toBe('Yo!');
-    expect(wrapper.hasClass('list-group')).toBe(true);
-    expect(wrapper.hasClass('list-group-flush')).toBe(true);
+    expect(screen.getByText('Yo!')).toHaveClass('list-group-flush');
```

Comment:
Maybe still include the assertion from `list-group` as well for all the variations. Just protection if someone does end up adding some sort of `if` statement to change that later.... but doesn't seem likely.